eox_core/api/v1/permissions.py

```diff
@@ -18,14 +19,16 @@ def load_permissions():
     if settings.EOX_CORE_LOAD_PERMISSIONS:
         try:
             content_type = ContentType.objects.get_for_model(User)
-            obj, created = Permission.objects.get_or_create(  # pylint: disable=unused-variable
+            Permission.objects.get_or_create(
                 codename='can_call_eox_core',
                 name='Can access eox-core API',
                 content_type=content_type,
             )
-        except ProgrammingError:
-            # This code runs when the app is loaded, if a migration has not been done a ProgrammingError exception is raised
-            # we are bypassing those cases to let migrations run smoothly.
+        except (ProgrammingError, ImproperlyConfigured):
+            # This code runs when the app is loaded, if a migration has not been done a
+            # ProgrammingError or ImproperlyConfugured exception is raised. We are
```

Comment:
There is a typo in ImproperlyConfigured. Also, the current comment implies that an ImproperlyConfigured error occurs when migrations have not been run, but usually this error indicates an issue with the db configuration rather than with pending migrations. 

What do you think of this suggestion?
```suggestion
# This code runs when the app is loaded. If a migration has not been done, a
# ProgrammingError is raised. We are bypassing this case to let migrations run smoothly.
# The ImproperlyConfigured exception typically indicates a configuration issue rather than
# a migration issue. This error handling ensures that the Open edX image can be built without
# interruption.
```

Reply:
Yes, you're right. The comment you propose is better. Thanks!
https://github.com/eduNEXT/eox-core/commit/d1ddc87c01d014da9d381886a0db0e032390eca4